hal/architecture/ESP8266/MyHwESP8266.cpp

```diff
@@ -29,7 +30,9 @@ bool hwInit(void)
 #endif
 #endif
 	EEPROM.begin(EEPROM_size);
-	return true;
+	// register _process() to be called at most every 1us,
+	// at every loop() or yield()
+	return schedule_recurrent_function_us([](){ _process(); return true; }, 1);
```

Comment:
The build system doesn't like the style of this line. Here is the patch suggested by the build system:
```
diff --git a/hal/architecture/ESP8266/MyHwESP8266.cpp b/hal/architecture/ESP8266/MyHwESP8266.cpp
index c9f7d2b..d76e2cf 100644
--- a/hal/architecture/ESP8266/MyHwESP8266.cpp
+++ b/hal/architecture/ESP8266/MyHwESP8266.cpp
@@ -32,7 +32,10 @@ bool hwInit(void)
 	EEPROM.begin(EEPROM_size);
 	// register _process() to be called at most every 1us,
 	// at every loop() or yield()
-	return schedule_recurrent_function_us([](){ _process(); return true; }, 1);
+	return schedule_recurrent_function_us([]() {
+		_process();
+		return true;
+	}, 1);
 }
 
 void hwReadConfigBlock(void *buf, void *addr, size_t length)
```

Reply:
Thanks !
Style is now fixed

Reply:
CI still does not send email but I now can read its reports:
commits are rebased, squashed and forced-push.
Let's see